src/SprykerEco/Yves/FactFinderWebComponents/Form/WebComponentConfigToTwigConfigTransformer.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Yves\FactFinderWebComponents\Form;
+
+use Symfony\Component\Form\DataTransformerInterface;
+
+class WebComponentConfigToTwigConfigTransformer implements DataTransformerInterface
+{
+    public const ITEMS_PARAMETER = 'items';
+
+    /**
+     * @inheritdoc
+     *
+     * @param mixed $value
+     *
+     * @return mixed|void
+     */
+    public function transform($value)
+    {
+        $result = [];
+
+        if (is_array($value) == false) {
```

Comment:
I think you can may not use comparison here, just `!is_array($value)` or at least use strict comparison.

Reply:
I added a strict comparison to avoid nested if statement.